File: basic_c/factor.c
#include<stdio.h>

int main(){
    int n,i;
    printf("enter a number: ");
    scanf("%d",&n);
    for ( i = 1; i < n+1; i++)
    {
        if (n%i==0){
            printf("%d",&i);
        }
    }
    
}

File: basic_go/factor.go
package main

import "fmt"

func main() {
	var n int
	fmt.Println("enter a number:")
	fmt.Scanf("%d", &n)
	for i := 1; i < n+1; i++ {
		if n%i == 0 {
			fmt.Println(i)
		}
	}

}
